tests/Integration/SDK/SpanLimitsBuilderTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Integration\SDK\Trace;
```

Comment:
The namespace is `OpenTelemetry\Tests\Integration\SDK\Trace`, but the directory is `tests/Integration/SDK/`
Please, change one or the other.

(This is not super important as the tests are not loaded/resolved via the namespace like library classes, but it makes for better consitency. It's also not super important which option you choose, as the integration tests do not necessarily have to match library classes)